res/res_pjsip_outbound_registration.c

```diff
@@ -209,6 +209,9 @@
 				<configOption name="support_outbound">
 					<synopsis>Enables advertising SIP Outbound support (RFC5626) for outbound REGISTER requests.</synopsis>
 				</configOption>
+				<configOption name="user_agent">
+					<synopsis>Overrides the User-Agent-Header that should be used for outbound REGISTER requests.</synopsis>
```

Comment:
What is the use of this versus doing it globally? Is it really only applicable to outbound registration and not calls as well? (Are you going to be making another review to do it for calls)

Reply:
We've discussed this a while ago at https://community.asterisk.org/t/setting-useragent-header-based-on-endpoint/94583/3 (and I wanted to submit this patch for a while but never realized figured out the old submission process). I have it ready for and endpoint aswell, which would cover calls I think, I just wanted to get this reviewed first as I was not sure at all about the correctness (seems like my concerns were valid and I've made various mistakes).